setup.py

```diff
@@ -5,6 +5,11 @@
 
 from setuptools import setup
 
+# Workaround for http://bugs.python.org/issue8876, see
+# http://bugs.python.org/issue8876#msg208792
+# This can be removed when using Python 2.7.9 or later:
+# https://hg.python.org/cpython/raw-file/v2.7.9/Misc/NEWS
+del os.link
```

Comment:
This looks dangerously fragile and will likely break other stuff... We should at least guard it by `if using vagrant`. Your entire comment from above should be copy&pasted into the source code. Please also reference PR number (#292) to make it even simpler to backtrack the rationale.